internal/app/go-filecoin/node/node.go

```diff
@@ -416,13 +416,13 @@ func (node *Node) SetupMining(ctx context.Context) error {
 func registeredProofsFromSectorSize(ss abi.SectorSize) (registeredSealProof abi.RegisteredProof, registeredPoStProof abi.RegisteredProof, err error) {
 	switch ss {
 	case constants.DevSectorSize:
-		return constants.DevRegisteredPoStProof, constants.DevRegisteredSealProof, nil
+		return constants.DevRegisteredWindowPoStProof, constants.DevRegisteredSealProof, nil
```

Comment:
This looks like it was already wrong, but the order of values here doesn't match the names of the out parameters. Please check the call sites and fix up one way or the other.